### ISSUE TITLE
Adding dependency's dependencies.

### ISSUE THREAD

**Issue by bebeli555** [GradleUp/shadow#596]
So when adding an dependency to the shadow jar. Unlike in eclipse where it automatically downloads all the libraries required for the dependency to work. Would this be possible for shadow too? So when adding an dependency to the shadow jar it would automatically add all the other libraries the dependency needs to work like in eclipse? Its quite a hassle adding like 20 libraries one by one just for the single dependency to function properly.

**Comment by Goooler** [comment]
Thy with `api group:artifact:version`
